WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -404,7 +407,114 @@ extension AztecPostViewController : Aztec.FormatBarDelegate
 
 
     func toggleLink() {
-        richTextView.toggleLink(range: richTextView.selectedRange, params: [String : AnyObject]())
+        var linkTitle = ""
```

Comment:
The following looks like a lot of new code for a swift 2.3 migration, can we split that into a separate PR? I'm not sure why it's here.